ERCS/erc-7787.md

```diff
@@ -0,0 +1,204 @@
+---
+eip: 7787
+title: Soulbound Degradable Governance
+description: DAO governance where voting power is non-transferable and decays over time without active contributions.
+author: Guilherme Neves (@0xneves), Rafael Castaneda <rafaelcastaneda@gmail.com>
+discussions-to: https://ethereum-magicians.org/t/erc-7787-soulbound-degradable-governance/21326
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-10-15
+requires: 20
+---
+
+## Abstract
+
+This proposal introduces the Soulbound Degradable Governance (SDG) standard, where governance power should be granted as non-transferable tokens that decay over time unless renewed through participation. SDG enables young DAOs to implement merit-based governance by detaching governance power from economic power while on early stages of development.
+
+## Motivation
+
+Traditional DAO governance models rely heavily on economic tokens, where voting power is proportional to token holdings. While effective for some use cases, this model risks concentrating power among wealthy members, leading to plutocracy and discouraging participation from smaller stakeholders. Furthermore, it fosters a treasury-centric culture that attracts contributors primarily focused on financial gain, rather than long-term governance or community well-being. 
+
+Young DAOs, in particular, need governance models that incentivize active contributions without relying on economic power. This proposal addresses these issues by detaching governance power from economic power and ensuring political power decays if not maintained through ongoing participation. This approach creates a merit-based structure that reflects continuous involvement and reduces the risk of early-stage centralization or dependent on heavy inflationary policies.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in RFC 2119 and RFC 8174.
+
+This system MUST operate with two distinct tokens types, one representing **political power** and another representing **economic power**:  
+
+1. The political power token SHOULD be non-transferable with over-time decayment. 
+
+2. The economic power token supports liquidity and trade, providing the financial utility needed for the DAO’s operations and is RECOMMENDED to be a standard [ERC-20](./eip-20.md) token.
+
+The implementer of this standard MUST:
+
+1. Override the `transfer(...)` function for the governance token to block transfers between addresses. 
```

Comment:
It is unclear whether the "governance token" here refers to the "political power" token or the "economic power" token. I'd guess that it's political, but it's better to be clear.